app/web/readme.md

```diff
@@ -1,16 +1,73 @@
 # Couchers web frontend
 
-[![Powered by Vercel](https://www.datocms-assets.com/31049/1618983297-powered-by-vercel.svg)](https://vercel.com?utm_source=couchers-org&utm_campaign=oss)
+This is the react/nextjs web frontend for Couchers.org. We are using Typescript with [React Query](https://react-query.tanstack.com/) for data fetching and [Material UI](https://material-ui.com/) for components.
 
-This is the react/nextjs web frontend for couchers.org. We are using Typescript with [React Query](https://react-query.tanstack.com/) for data fetching and [Material UI](https://material-ui.com/) for components.
+Communication with the backend is via [protobuf messages](https://github.com/protocolbuffers/protobuf-javascript) over [grpc-web](https://github.com/grpc/grpc-web). You can find some helpful documentation on [protobuf messages in javascript here](https://protobuf.dev/protobuf-javascript/).
 
-Communication with the backend is via [protobuf messages](https://github.com/protocolbuffers/protobuf/tree/master/js) and [grpc-web](https://github.com/grpc/grpc-web). You can find some helpful documentation on [protobuf messages in javascript here](https://developers.google.com/protocol-buffers/docs/reference/javascript-generated).
+## Quick Start
+
+We recommend `nvm` (the [node version manager](https://github.com/nvm-sh/nvm)). You can install it with:
+
+```sh
+wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash
+```
+
+**You will need to restart your terminal before `nvm` becomes available.**
+
+<details>
+<summary>More on deps</summary>
+
+You need `nodejs` v20 and the `yarn` package manager. The recommended way to do this is to use NVM as above.
+
+If you don't want `nvm` to be run when you open your shell, you can add `--no-use` in your shell's `rc` file, just before where it says `# This loads nvm`. You then need to run `nvm use` every time you enter the `app/web` folder to start developing.
+
```

Comment:
I just did `yarn start` is that no longer correct?

Reply:
It’s valid but if you install nvm it’ll run every time you start your terminal. For me that causes a really annoying slow-down… hence this comment

Reply:
Ah gotcha, maybe another reason against nvm lol.